### DISCUSSION TITLE
Hotfix for some important issues

### DISCUSSION THREAD

**Discussion by KosmX** [KosmX/jneedle#49] [Announcements]
- Fix launchwrapper relocation, it should fix classpath collision
- fix java 1.8 compatibility using LLJzip fork https://github.com/KosmX/LL-Java-Zip/tree/master (until fix is merged)
- 2.0 branch: fix multiple jar archive reading issues

## GUI still needs java 11+, CLI and launch-wrapper should be java 1.8 compatible

<hr /><em>This discussion was created from the release <a href='https://github.com/KosmX/jneedle/releases/tag/1.1.0-local'>Hotfix for some important issues</a>.</em>

**Comment by Griefed** [comment]
Heyooo,
I've recently integrated jNeedle into ServerPackCreator. However, due to jNeedle not yet being on maven, I made use of jitpack. There appears to be an error with the manifest, though, as jitpack can only provide version up to 1.0.1.
Once you try to use 1.1.0, it throws a fit:

- Build Log from jitpack: https://jitpack.io/com/github/KosmX/jneedle/1.1.0-local/build.log
- Overview: https://jitpack.io/#KosmX/jneedle

I'd like to make use of 1.1.0, as I hope the JAR reading fixes will prevent a deadlock currently occuring with some mods.

Once I encounter `2024-08-26T12:24:01,616  WARN [DefaultDispatcher-worker-7] (JvmZipReaderStrategy.java:148) - Central-Directory-File-Header's offset[387626] to Local-File-Header does not match the Local-File-Header magic!` with a mod it won't continue, effectively being stuck.

Any ideas on this?

EDIT: The mod which causes this deadlock: https://www.curseforge.com/minecraft/mc-mods/rrls/files/5423270

Cheers,
Griefed
